data/json/items/comestibles/raw_fruit.json

```diff
@@ -675,20 +675,20 @@
     "type": "COMESTIBLE",
     "id": "melon_chunks",
     "name": { "str": "handful of cantaloupe melon chunks", "str_pl": "handfuls of cantaloupe melon chunks" },
-    "weight": "160 g",
-    "color": "brown",
+    "weight": "138 g",
+    "color": "light_red",
     "spoils_in": "3 days",
     "comestible_type": "FOOD",
     "symbol": "%",
-    "quench": 8,
-    "calories": 54,
+    "quench": 25,
+    "calories": 47,
     "description": "A handful of cantaloupe melon chunks.  A great snack by itself or in a fruit salad.",
-    "price": 139,
-    "price_postapoc": 60,
+    "price": 21,
+    "price_postapoc": 25,
     "material": [ "fruit" ],
-    "volume": "250 ml",
+    "volume": "275 ml",
```

Comment:
Please change that back. I specifically kept all 'handfuls' as 250ml (1 cup) for ease of calculation.